powerdnsadmin/routes/index.py

```diff
@@ -674,7 +683,14 @@ def register():
                 if result and result['status']:
                     if Setting().get('verify_user_email'):
                         send_account_verification(email)
-                    return redirect(url_for('index.login'))
+                    if Setting().get('otp_force') and Setting().get('otp_field_enabled'):
+                        login_user(user)
```

Comment:
This line can be removed, simply replace `current_user` with `user` in the following two lines.